agent/lib/kontena/workers/service_pod_worker.rb

```diff
@@ -273,5 +334,12 @@ def sync_state_to_master(current_state, error = nil)
     def log_service_pod_event(type, data, severity = Logger::INFO)
       super(service_pod.service_id, service_pod.instance_number, type, data, severity)
     end
+
+    private
+
+    def shutdown
+      @restart_backoff_timer.cancel if @restart_backoff_timer
```

Comment:
If it is needed, then do we need to add something similar to the other actors that use timers as well?